server/cluster.go

```diff
@@ -119,11 +121,10 @@ func (c *RaftCluster) start() error {
 	var classifier namespace.Classifier
 	if c.s.cfg.EnableNamespace {
 		log.Infoln("use namespace classifier.")
-		nsInfo := c.cachedCluster.namespacesInfo
-		if err := nsInfo.loadNamespaces(c.s.kv, kvRangeLimit); err != nil {
+		classifier, err = newTableNamespaceClassifier(core.DefaultTableIDDecoder, c.s.kv, c.s.idAlloc)
```

Comment:
I think we can remove the decoder argument here too. We always use the default decoder for the table classifier.

Reply:
Agree. But I prefer to keep this PR's intent accurate. We can do it later.